chain-prometheus-exporter/src/main.rs

```diff
@@ -60,7 +60,8 @@ struct App {
     #[clap(
         long = "account",
         help = "List of account addresses to monitor.",
-        env = "CHAIN_PROMETHEUS_EXPORTER_ACCOUNTS"
+        env = "CHAIN_PROMETHEUS_EXPORTER_ACCOUNTS",
+        value_delimiter = ','
```

Comment:
Please update the changelog and version.

Reply:
Which changelog and version are we talking about here?

Reply:
There are no changelog in the `chain-prometheus-exporter`

Reply:
Well then make one.

Version is in Cargo.toml file. This is a bugfix so it gets a patch version bump.